src/it/scala/tech/cryptonomic/conseil/client/EndpointClient.scala

```diff
@@ -153,35 +161,29 @@ object DataEndpointsClientProbe {
         endpoint,
         `Content-Type`(MediaType.application.json),
         Accept(MediaType.application.json),
-        Header("apiKey", "hooman")
+        Header("apiKey", apiKey)
       )
       client.expect[Json](req).attempt.map {
-        case Right(json) =>
-          Either.cond(
-            json == expected,
-            right = json,
-            left = new Exception(
-              s"Failed to match on $endpoint: expected \n${expected.spaces2} \n but found \n ${json.spaces2}"
-            )
-          )
+        case Right(received) =>
+          Diff[Json].apply(received, expected) match {
```

Comment:
Notes: switching to software-mill's diff allows to receive a better-detailed log of how two complex data-structure differ between expectations and test results.